docs/core/install/linux-package-mixup.md

```diff
@@ -111,15 +116,18 @@ If your distribution provides .NET packages, it's recommended that you use that
         sudo apt-get update && sudo apt-get install -y dotnet7
         ```
 
-03. **I need a recent version of .NET that's not provided by the Linux distribution repositories.**
+03. **I need a recent version of .NET that's not provided by the Linux distribution repositories and I still want to use the distribution repository for supported .NET packages.**
```

Comment:
The Ubuntu and Fedora steps seem to be doing different things?

The Fedora steps uses Microsoft repositories for all .NET versions (but .NET packages only) while the Ubuntu steps use Microsoft repositories only for some (?) .NET packages.

Reply:
Can you clarify what is going on here? When you add the Microsoft repository feed to an Ubuntu system, it seems to automatically take precedence over the built-in feed. Is that not the case with the commands for Fedora? I sort of wrote Item 5 with that assumption, that you didn't need to set priority on the repo.

Reply:
First, a question: Is the "recent version" here referring to brand new major .NET versions (eg, 7.0 vs 8.0) or does it also include SDK bumps (7.0.102 vs 7.0.200)?

The Fedora steps, from what I can tell, do this:
- Remove all .NET-related packages
- Make the Microsoft repository a higher-priority repository. This makes `dnf` prefer packages from there if packages are available from multiple repositories, even if the package is at a lower version. The command doesn't restrict the priority to any particular set of packages. So all packages from the Microsoft repository are preferred over Fedora packages
- Install .NET 7 SDK

However, after running the commands, any/all packages from the Microsoft repository are preferred over Fedora's repository. So if a user later installs .NET 6 (`dnf install -y dotnet-sdk-6.0`) that will also use Microsoft's packages, even if .NET 6 is available in Fedora.

I tested this out to confirm this:

```
$ podman run -it registry.fedoraproject.org/fedora:37
# dnf install https://packages.microsoft.com/fedora/37/prod/packages-microsoft-prod.rpm
# echo 'priority=50' | sudo tee -a /etc/yum.repos.d/microsoft-prod.repo
# dnf install dotnet-sdk-7.0                                                                                                             
packages-microsoft-com-prod                                                                                                    15 kB/s | 3.0 kB     00:00     
Dependencies resolved.                                                                                                                                        
==============================================================================================================================================================
 Package                                          Architecture             Version                        Repository                                     Size 
==============================================================================================================================================================
Installing:                                                                                                                                                   
 dotnet-sdk-7.0                                   x86_64                   7.0.201-1                      packages-microsoft-com-prod                   133 M 
Installing dependencies:                                                                                                                                      
 aspnetcore-runtime-7.0                           x86_64                   7.0.3-1                        packages-microsoft-com-prod                   9.3 M 
 aspnetcore-targeting-pack-7.0                    x86_64                   7.0.3-1                        packages-microsoft-com-prod                   2.4 M 
 dotnet-apphost-pack-7.0                          x86_64                   7.0.3-1                        packages-microsoft-com-prod                   4.7 M 
 dotnet-host                                      x86_64                   7.0.3-1                        packages-microsoft-com-prod                    78 k 
 dotnet-hostfxr-7.0                               x86_64                   7.0.3-1                        packages-microsoft-com-prod                   180 k 
 dotnet-runtime-7.0                               x86_64                   7.0.3-1                        packages-microsoft-com-prod                    30 M 
 dotnet-runtime-deps-7.0                          x86_64                   7.0.3-1                        packages-microsoft-com-prod                   8.4 k 
 dotnet-targeting-pack-7.0                        x86_64                   7.0.3-1                        packages-microsoft-com-prod                   3.8 M 
 libicu                                           x86_64                   71.1-2.fc37                    fedora                                         10 M 
 netstandard-targeting-pack-2.1                   x86_64                   2.1.0-1                        packages-microsoft-com-prod                   2.1 M 
                                                                                                                                                              
Transaction Summary                                                                                                                                           
==============================================================================================================================================================
Install  11 Packages                                                                                                                                          
                                                                                                                                                              
Total download size: 196 M                                                                                                                                    
Installed size: 547 M                                                                                                                                         
Is this ok [y/N]: ^COperation aborted.                                                                                                                        
# dnf install dotnet-sdk-6.0                                                                                                             
Last metadata expiration check: 0:00:07 ago on Fri Mar  3 17:45:22 2023.                                                                                      
Dependencies resolved.                                                                                                                                        
==============================================================================================================================================================
 Package                                          Architecture             Version                        Repository                                     Size 
==============================================================================================================================================================
Installing:                                                                                                                                                   
 dotnet-sdk-6.0                                   x86_64                   6.0.406-1                      packages-microsoft-com-prod                   126 M 
Installing dependencies:                                                                                                                                      
 aspnetcore-runtime-6.0                           x86_64                   6.0.14-1                       packages-microsoft-com-prod                   8.7 M 
 aspnetcore-targeting-pack-6.0                    x86_64                   6.0.14-1                       packages-microsoft-com-prod                   2.0 M 
 dotnet-apphost-pack-6.0                          x86_64                   6.0.14-1                       packages-microsoft-com-prod                   4.7 M 
 dotnet-host                                      x86_64                   7.0.3-1                        packages-microsoft-com-prod                    78 k 
 dotnet-hostfxr-6.0                               x86_64                   6.0.14-1                       packages-microsoft-com-prod                   173 k 
 dotnet-runtime-6.0                               x86_64                   6.0.14-1                       packages-microsoft-com-prod                    29 M 
 dotnet-runtime-deps-6.0                          x86_64                   6.0.14-1                       packages-microsoft-com-prod                   3.9 k 
 dotnet-targeting-pack-6.0                        x86_64                   6.0.14-1                       packages-microsoft-com-prod                   3.1 M 
 libicu                                           x86_64                   71.1-2.fc37                    fedora                                         10 M 
 netstandard-targeting-pack-2.1                   x86_64                   2.1.0-1                        packages-microsoft-com-prod                   2.1 M 
                                                                                                                                                              
Transaction Summary                                                                                                                                           
==============================================================================================================================================================
Install  11 Packages                                                                                                                                          
                                                                                                                                                              
Total download size: 186 M                                                                                                                                    
Installed size: 518 M                                                                                                                                         
Is this ok [y/N]: ^COperation aborted.                                                                                                                        
```

Reply:
> I need a recent version of .NET that's not provided by the Linux distribution repositories and I still want to use the distribution repository for supported .NET packages.

I thought some more about this and I think this doesn't make much sense. The reason we started this entire package-mixup doc was that we found out that mixing .NET from different vendors would produce broken stuff and a bad user experience.

Shouldn't we remove this approach completely?

I tried out the steps for Ubuntu, and it fails in 22.10 and produces surprises on 22.04.

22.10

We can't install .NET 7 at all when Microsoft and Ubuntu repos are enabled:

```
root@332ca91cf4e9:/# apt update                                                                                                                               
Hit:1 http://security.ubuntu.com/ubuntu kinetic-security InRelease                                                                                            
Hit:2 http://archive.ubuntu.com/ubuntu kinetic InRelease                                                                                                      
Hit:3 http://archive.ubuntu.com/ubuntu kinetic-updates InRelease                                                                                              
Hit:4 http://archive.ubuntu.com/ubuntu kinetic-backports InRelease                                                                                            
Get:5 https://packages.microsoft.com/ubuntu/22.10/prod kinetic InRelease [10.5 kB]                                                                            
Get:6 https://packages.microsoft.com/ubuntu/22.10/prod kinetic/main armhf Packages [514 B]                                                                    
Get:7 https://packages.microsoft.com/ubuntu/22.10/prod kinetic/main amd64 Packages [26.2 kB]                                                                  
Get:8 https://packages.microsoft.com/ubuntu/22.10/prod kinetic/main arm64 Packages [2669 B]                                                                   
Fetched 39.8 kB in 1s (44.1 kB/s)                                                                                                                             
Reading package lists... Done                                                                                                                                 
Building dependency tree... Done                                                                                                                              
Reading state information... Done                                                                                                                             
2 packages can be upgraded. Run 'apt list --upgradable' to see them.                                                                                          
root@332ca91cf4e9:/# apt install dotnet-sdk-7.0                                                                                                               
Reading package lists... Done                                                                                                                                 
Building dependency tree... Done                                                                                                                              
Reading state information... Done                                                                                                                             
Some packages could not be installed. This may mean that you have                                                                                             
requested an impossible situation or if you are using the unstable                                                                                            
distribution that some required packages have not yet been created                                                                                            
or been moved out of Incoming.                                                                                                                                
The following information may help to resolve the situation:                                                                                                  
                                                                                                                                                              
The following packages have unmet dependencies:                                                                                                               
 netstandard-targeting-pack-2.1-7.0 : Conflicts: netstandard-targeting-pack-2.1                                                                               
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages. 
```

22.04:

We can install both Microsoft's SDK (7.0) and Ubuntu's SDK (6.0), but only 7.0 is picked up and available to users ...

```
# apt install dotnet-sdk-7.0                                
Reading package lists... Done                                                  
Building dependency tree... Done
Reading state information... Done                                                                                                                             
The following additional packages will be installed:                                                                                                          
  aspnetcore-runtime-7.0 aspnetcore-targeting-pack-7.0 dotnet-apphost-pack-7.0 dotnet-host dotnet-hostfxr-7.0 dotnet-runtime-7.0 dotnet-runtime-deps-7.0      
  dotnet-targeting-pack-7.0 libicu70 netstandard-targeting-pack-2.1                                                                                           
The following NEW packages will be installed:                                                                                                                 
  aspnetcore-runtime-7.0 aspnetcore-targeting-pack-7.0 dotnet-apphost-pack-7.0 dotnet-host dotnet-hostfxr-7.0 dotnet-runtime-7.0 dotnet-runtime-deps-7.0      
  dotnet-sdk-7.0 dotnet-targeting-pack-7.0 libicu70 netstandard-targeting-pack-2.1                                                                            0 upgraded, 11 newly installed, 0 to remove and 0 not upgraded.                                                                                               
Need to get 91.8 MB/142 MB of archives.                                                                                                                       
After this operation, 572 MB of additional disk space will be used.                                                                                           
Do you want to continue? [Y/n] y                                                                                                                              
Get:1 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-sdk-7.0 amd64 7.0.201-1 [91.8 MB]
Fetched 23.6 MB in 1min 20s (296 kB/s)
debconf: delaying package configuration, since apt-utils is not installed                                                                                     
Selecting previously unselected package libicu70:amd64.                                                                                                       
(Reading database ... 4883 files and directories currently installed.)         
Preparing to unpack .../00-libicu70_70.1-2_amd64.deb ...                       
Unpacking libicu70:amd64 (70.1-2) ...                                                                                                                         
Selecting previously unselected package dotnet-runtime-deps-7.0.               
Preparing to unpack .../01-dotnet-runtime-deps-7.0_7.0.3-1_amd64.deb ...                                                                                      
Unpacking dotnet-runtime-deps-7.0 (7.0.3-1) ...                                                                                                               
Selecting previously unselected package dotnet-host.                                                                                                          
Preparing to unpack .../02-dotnet-host_7.0.3-1_amd64.deb ...                                                                                                  
Unpacking dotnet-host (7.0.3-1) ...                                                                                                                           
Selecting previously unselected package dotnet-hostfxr-7.0.                                                                                                   
Preparing to unpack .../03-dotnet-hostfxr-7.0_7.0.3-1_amd64.deb ...                                                                                           
Unpacking dotnet-hostfxr-7.0 (7.0.3-1) ...                                                                                                                    
Selecting previously unselected package dotnet-runtime-7.0.                                                                                                   
Preparing to unpack .../04-dotnet-runtime-7.0_7.0.3-1_amd64.deb ...                                                                                           
Unpacking dotnet-runtime-7.0 (7.0.3-1) ...                                                                                                                    
Selecting previously unselected package aspnetcore-runtime-7.0.                                                                                               
Preparing to unpack .../05-aspnetcore-runtime-7.0_7.0.3-1_amd64.deb ...        
Unpacking aspnetcore-runtime-7.0 (7.0.3-1) ...                                                                                                                
Selecting previously unselected package dotnet-targeting-pack-7.0.                                                                                            
Preparing to unpack .../06-dotnet-targeting-pack-7.0_7.0.3-1_amd64.deb ...     
Unpacking dotnet-targeting-pack-7.0 (7.0.3-1) ...                              
Selecting previously unselected package aspnetcore-targeting-pack-7.0.                                                                                        Preparing to unpack .../07-aspnetcore-targeting-pack-7.0_7.0.3-1_amd64.deb ...                                                                                
Unpacking aspnetcore-targeting-pack-7.0 (7.0.3-1) ...                                                                                                         
Selecting previously unselected package dotnet-apphost-pack-7.0.                                                                                              
Preparing to unpack .../08-dotnet-apphost-pack-7.0_7.0.3-1_amd64.deb ...                                                                                      
Unpacking dotnet-apphost-pack-7.0 (7.0.3-1) ...                                                                                                               
Selecting previously unselected package netstandard-targeting-pack-2.1.                                                                                       
Preparing to unpack .../09-netstandard-targeting-pack-2.1_6.0.113-0ubuntu1~22.04.1_amd64.deb ...                                                              Unpacking netstandard-targeting-pack-2.1 (6.0.113-0ubuntu1~22.04.1) ...                                                                                       
Selecting previously unselected package dotnet-sdk-7.0.                                                                                                       
Preparing to unpack .../10-dotnet-sdk-7.0_7.0.201-1_amd64.deb ...                                                                                             
Unpacking dotnet-sdk-7.0 (7.0.201-1) ...                                                                                                                      
Setting up dotnet-host (7.0.3-1) ...                                                                                                                          
Setting up netstandard-targeting-pack-2.1 (6.0.113-0ubuntu1~22.04.1) ...                                                                                      
Setting up dotnet-targeting-pack-7.0 (7.0.3-1) ...                                                                                                            
Setting up dotnet-apphost-pack-7.0 (7.0.3-1) ...                               
Setting up dotnet-hostfxr-7.0 (7.0.3-1) ...                                    
Setting up libicu70:amd64 (70.1-2) ...                                         
Setting up dotnet-runtime-deps-7.0 (7.0.3-1) ...                                                                                                              
Setting up aspnetcore-targeting-pack-7.0 (7.0.3-1) ...                         
Setting up dotnet-runtime-7.0 (7.0.3-1) ...                                    
Setting up aspnetcore-runtime-7.0 (7.0.3-1) ...                                                                                                               
Setting up dotnet-sdk-7.0 (7.0.201-1) ...                                      
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...                                                                                                     
root@c09ce4c3879a:/# dotnet --info                                                                                                                            
.NET SDK:                                                                                                                                                     
 Version:   7.0.201                                                                                                                                           
 Commit:    68f2d7e7a3                                                                                                                                        
                                                                                                                                                              
Runtime Environment:                                                                                                                                          
 OS Name:     ubuntu                                                                                                                                          
 OS Version:  22.04                                                                                                                                           
 OS Platform: Linux                                                            
 RID:         ubuntu.22.04-x64                                                                                                                                
 Base Path:   /usr/share/dotnet/sdk/7.0.201/                                                                                                                  
                                                                               
Host:                                                                          
  Version:      7.0.3                                                                                                                                           Architecture: x64                                                                                                                                           
  Commit:       0a2bda10e8                                                                                                                                    
                                                                                                                                                              
.NET SDKs installed:                                                                                                                                          
  7.0.201 [/usr/share/dotnet/sdk]                                                                                                                             
                                                                                                                                                              
.NET runtimes installed:                                                                                                                                        Microsoft.AspNetCore.App 7.0.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]                                                                          
  Microsoft.NETCore.App 7.0.3 [/usr/share/dotnet/shared/Microsoft.NETCore.App]                                                                                
                                                                                                                                                              
Other architectures found:                                                                                                                                    
  None                                                                                                                                                        
                                                                                                                                                              
Environment variables:                                                                                                                                        
  Not set                                                                      
                                                                               
global.json file:                                                              
  Not found                                                                                                                                                   
                                                                               
Learn more:                                                                    
  https://aka.ms/dotnet/info                                                                                                                                  
                                                                               
Download .NET:                                                                 
  https://aka.ms/dotnet/download            
root@c09ce4c3879a:/# apt install dotnet-sdk-6.0                                                                                                               
Reading package lists... Done                                                                                                                                 
Building dependency tree... Done                                                                                                                              
Reading state information... Done                                                                                                                             
The following additional packages will be installed:                                                                                                          
  aspnetcore-runtime-6.0 aspnetcore-targeting-pack-6.0 dotnet-apphost-pack-6.0 dotnet-hostfxr-6.0 dotnet-runtime-6.0 dotnet-targeting-pack-6.0                
  liblttng-ust-common1 liblttng-ust-ctl5 liblttng-ust1 libnuma1 libunwind8                                                                                    
The following NEW packages will be installed:                                  
  aspnetcore-runtime-6.0 aspnetcore-targeting-pack-6.0 dotnet-apphost-pack-6.0 dotnet-hostfxr-6.0 dotnet-runtime-6.0 dotnet-sdk-6.0                           
  dotnet-targeting-pack-6.0 liblttng-ust-common1 liblttng-ust-ctl5 liblttng-ust1 libnuma1 libunwind8                                                          
0 upgraded, 12 newly installed, 0 to remove and 0 not upgraded.                
Need to get 126 MB of archives.                                                
After this operation, 488 MB of additional disk space will be used.                                                                                           Do you want to continue? [Y/n] y                                                                                                                              
Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 dotnet-targeting-pack-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [2248 kB]                        
Get:2 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 dotnet-sdk-6.0 amd64 6.0.406-1 [86.7 MB]                                              
Get:3 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 dotnet-hostfxr-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [154 kB]                                
Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust-common1 amd64 2.13.1-1ubuntu1 [27.1 kB]                                                  
Get:5 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnuma1 amd64 2.0.14-3ubuntu2 [22.5 kB]                                                              
Get:6 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust-ctl5 amd64 2.13.1-1ubuntu1 [77.9 kB]                                                     Get:7 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblttng-ust1 amd64 2.13.1-1ubuntu1 [190 kB]                                                          
Get:8 http://archive.ubuntu.com/ubuntu jammy/main amd64 libunwind8 amd64 1.3.2-2build2 [54.5 kB]                                                              
Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 dotnet-runtime-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [23.7 MB]                               
Get:10 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 aspnetcore-runtime-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [8092 kB]                          
Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 dotnet-apphost-pack-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [3737 kB]                         
Get:12 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 aspnetcore-targeting-pack-6.0 amd64 6.0.113-0ubuntu1~22.04.1 [1455 kB]                   
Fetched 126 MB in 3min 16s (645 kB/s)                                                                                                                         
debconf: delaying package configuration, since apt-utils is not installed 
Selecting previously unselected package dotnet-sdk-6.0.                                                                                               [0/1389]
Preparing to unpack .../11-dotnet-sdk-6.0_6.0.406-1_amd64.deb ...                                                                                             
Unpacking dotnet-sdk-6.0 (6.0.406-1) ...                                                                                                                      
Setting up liblttng-ust-common1:amd64 (2.13.1-1ubuntu1) ...                    
Setting up dotnet-targeting-pack-6.0 (6.0.113-0ubuntu1~22.04.1) ...            
Setting up libunwind8:amd64 (1.3.2-2build2) ...                                                                                                               
Setting up aspnetcore-targeting-pack-6.0 (6.0.113-0ubuntu1~22.04.1) ...        
Setting up dotnet-hostfxr-6.0 (6.0.113-0ubuntu1~22.04.1) ...                                                                                                  
Setting up libnuma1:amd64 (2.0.14-3ubuntu2) ...                                                                                                               
Setting up dotnet-apphost-pack-6.0 (6.0.113-0ubuntu1~22.04.1) ...                                                                                             
Setting up liblttng-ust-ctl5:amd64 (2.13.1-1ubuntu1) ...                                                                                                      
Setting up liblttng-ust1:amd64 (2.13.1-1ubuntu1) ...                                                                                                          
Setting up dotnet-runtime-6.0 (6.0.113-0ubuntu1~22.04.1) ...                                                                                                  
Setting up aspnetcore-runtime-6.0 (6.0.113-0ubuntu1~22.04.1) ...                                                                                              
Setting up dotnet-sdk-6.0 (6.0.406-1) ...                                                                                                                     
This software may collect information about you and your use of the software, and send that to Microsoft.                                                     
Please visit http://aka.ms/dotnet-cli-eula for more information.                                                                                              
Welcome to .NET!                                                                                                                                              
---------------------                                                                                                                                         
Learn more about .NET: https://aka.ms/dotnet-docs                              
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli-docs                                                                        
                                                                                                                                                              
Telemetry                                                                      
---------                                                                      
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.                                         
                                                                                                                                                              
Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry                                                                                 
                                                                                                                                                              
Configuring...                                                                                                                                                
--------------                                                                                                                                                
A command is running to populate your local package cache to improve restore speed and enable offline access. This command takes up to one minute to complete and only runs once.                                                                                                                                           
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...                                                                                                        
root@c09ce4c3879a:/# dotnet --info                                                                                                                            
.NET SDK:                                                                                                                                                     
 Version:   7.0.201                                                                                                                                           
 Commit:    68f2d7e7a3                                                                                                                                        
                                                                                                                                                              
Runtime Environment:                                                           
 OS Name:     ubuntu                                                           
 OS Version:  22.04                                                            
 OS Platform: Linux                                                                                                                                           
 RID:         ubuntu.22.04-x64                                                 
 Base Path:   /usr/share/dotnet/sdk/7.0.201/                                   
                                                                                                                                                              
Host:                                                                          
  Version:      7.0.3                                                          
  Architecture: x64                                                            
  Commit:       0a2bda10e8                                                     
                                                                               
.NET SDKs installed:                                                           
  6.0.406 [/usr/share/dotnet/sdk]                                                                                                                             
  7.0.201 [/usr/share/dotnet/sdk]                                              
                                                                                                                                                              
.NET runtimes installed:                                                       
  Microsoft.AspNetCore.App 7.0.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]                                                                          
  Microsoft.NETCore.App 7.0.3 [/usr/share/dotnet/shared/Microsoft.NETCore.App]                                                                                
                                                                                                                                                              
Other architectures found:                                                     
  None                                                                         
                                                                                                                                                              
Environment variables:                                                                                                                                        
  Not set                                                                                                                                                     
                                                                                                                                                              
global.json file:                                                                                                                                             
  Not found                                                                                                                                                   
                                                                                                                                                              
Learn more:                                                                                                                                                   
  https://aka.ms/dotnet/info                                                                                                                                  
                                                                                                                                                              
Download .NET:                                                                                                                                                
  https://aka.ms/dotnet/download

Reply:
> Shouldn't we remove this approach completely?

I agree. We do not need to document all creative ways to mix and match. The documentation should be focused on the few simple options that are guaranteed to work.

Reply:
@jkotas @omajid Can we clarify what scenarios we want to cover? If it's minimal, I think it would just be this few. Note that the "Uninstall .NET" step is there to get them back to a clean slate, and then choose which repository to from then on install .NET.

01. I want to use the distro feed and ignore Micrsoft's .NET, but I need the MS repository for other packages.

    - Uninstall .NET
    - Configure the MS repository to ignore .NET
    - Reinstall .NET

01. I want to use the distro feed and ignore all of MS repository packages

    - Uninstall .NET
    - Remove the MS repository
    - Reinstall .NET

01. I need versions of .NET that aren't provided by my distro feed

    - Uninstall .NET
    - Configure the distro repository to ignore .NET
    - Reinstall.NET
 
Do we want to cover the more complex mix-and-match scenarios?

Reply:
This list looks reasonable to me.

Reply:
Resolving this conversation, I've rewritten the content.